### PR TITLE
Generate LST files by default

### PR DESCRIPTION
**Pull request by ppannuto**

### CODE REVIEW

**Comment by alevy** [comment]
Didn't we decide against this some time ago? Or was that just for the kernel?

**Comment by brghena** [comment]
I believe the only reason we were against this before was due to how long it took on mac. As an experiment, I tried building LST files before and after running the fix that Pat found.

To put these in perspective, "sensors" takes 1.5 s to `make` and "duktape" takes 7.9 s to `make`

## Before
userland/examples/sensors: 1m 14s
[duktape](https://github.com/cxbrooks/tock/tree/cxbrooks/duktape3): userland/examples/duktape: 3m 1s

## After
userland/examples/sensors: 0.624s
[duktape](https://github.com/cxbrooks/tock/tree/cxbrooks/duktape3): userland/examples/duktape: 18s

Note that the "duktape" app.lst is a 25 MB file with 700000 lines.

**Comment by bradjc** [comment]
I thought that the fix was in the Makefile. Given that it is a note in the readme that no one will see, this seems to go against make tock easy to use. Are you really going to use an embedded OS that takes 74 seconds just to build an app called "sensors"?

**Comment by alevy** [comment]
I think we are at least all in agreement, though, that the changes to the README are good --- i.e. explaining how to make generating lst files quickly on Mac is a positive addition. The disagreement is about whether to generate them by default, right?

**Comment by ppannuto** [comment]
Yep. I'll split that out, one sec

**Comment by ppannuto** [comment]
Done

File: doc/Getting_Started.md [hunk 1]
```diff
@@ -57,6 +57,16 @@ or with [MacPorts](https://www.macports.org/):
 $ port install arm-none-eabi-gcc
 ```
 
+###### Heads Up!
+
+By default Tock generates a listings (disassembly) when it builds apps. Some
```
Comment (bradjc) [inline doc/Getting_Started.md]: This needs to be updated.

Comment (ppannuto) [inline doc/Getting_Started.md]: good catch; fixed

**Pull request merged**